packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -292,175 +297,187 @@ export const initializeChugSplash = async (
+  // 2. If the owner is the multisig and we're deploying on a live network then we have to use the gnosis safe ethers adapter (which we have not implemented yet).
+  // 3. We also allow the user to specify a different owner via process.env.OWNER_PRIVATE_KEY. This is useful for testing on live networks without using the multisig.
+  //    In this case, we need to create a signer using the OWNER_PRIVATE_KEY and use that.
+  let signer: ethers.Signer
+
+  // If deploying on a live network and the target owner is the multisig, then throw an error because
+  // we have not setup the safe ethers adapter yet.
+  if (
+    (await isLiveNetwork(provider)) &&
+    getOwnerAddress() === OWNER_MULTISIG_ADDRESS
+  ) {
+    // TODO: use the safe ethers adapter to send multisig transactions
+    throw new Error(
+      'Cannot run multisig transactions on a live network, please setup the safe ethers adapter first https://www.npmjs.com/package/@safe-global/safe-ethers-adapters'
     )
+  } else {
+    // if target owner is multisig, then use an impersonated multisig signer
+    if (getOwnerAddress() === OWNER_MULTISIG_ADDRESS) {
+      console.log('signer is owner multisig')
+      signer = await getImpersonatedSigner(OWNER_MULTISIG_ADDRESS, provider)
+    } else {
+      // if target owner is not multisig, then use the owner signer
+      // OWNER_PRIVATE_KEY will always be defined if the OWNER_ADDRESS is not the OWNER_MULTISIG_ADDRESS
+      signer = new ethers.Wallet(process.env.OWNER_PRIVATE_KEY!, provider)
+    }
 
-    // Fund the multisig.
+    // Fund the signer
     await (
       await deployer.sendTransaction({
-        to: OWNER_MULTISIG_ADDRESS,
+        to: await signer.getAddress(),
         value: ethers.utils.parseEther('0.1'),
       })
     ).wait()
+  }
+
+  logger?.info('[ChugSplash]: adding the initial ChugSplashManager version...')
+
+  if (
+    (await ChugSplashRegistry.managerImplementations(
+      ChugSplashManager.address
+    )) === false
+  ) {
+    await (
+      await ChugSplashRegistry.connect(signer).addVersion(
+        ChugSplashManager.address,
+        await getGasPriceOverrides(provider)
+      )
+    ).wait()
+  }
+
+  logger?.info('[ChugSplash]: added the initial ChugSplashManager version')
 
+  logger?.info('[ChugSplash]: assigning executor roles...')
+  for (const executor of executors) {
     if (
-      (await ChugSplashRegistry.managerImplementations(
-        ChugSplashManager.address
+      (await ManagedService.hasRole(
+        utils.defaultAbiCoder.encode(['bytes32'], [EXECUTOR_ROLE]),
+        executor
       )) === false
     ) {
-      await (
-        await ChugSplashRegistry.connect(multisig).addVersion(
-          ChugSplashManager.address,
-          await getGasPriceOverrides(provider)
-        )
-      ).wait()
+      await ManagedService.connect(signer).grantRole(
+        utils.defaultAbiCoder.encode(['bytes32'], [EXECUTOR_ROLE]),
```

Comment:
We can change `utils.defaultAbiCoder.encode(['bytes32'], [EXECUTOR_ROLE])` to just `EXECUTOR_ROLE`